framework/src/media/audio/audio_manager.h

```diff
@@ -369,6 +369,6 @@ audio_manager_result_t set_input_audio_volume(uint8_t volume);
 audio_manager_result_t set_output_audio_volume(uint8_t volume);
 
 #if defined(__cplusplus)
-}								/* extern "C" */
+} /* extern "C" */
```

Comment:
I already applied our cformatter.sh to audio_manager.c and audio_manager.h several times.
The existing cformatter is wrong...?